src/functions.php

```diff
@@ -227,11 +228,12 @@ function after(string $task, $do)
  * When called multiple times for a task, previous fail() definitions will be overridden.
  *
  * @param string $task The task which need to fail so $that should be run.
- * @param string $do The task to be run.
+ * @param string|callable $do The task to be run.
+ * @phpstan-param string|callable(): void $do
  *
  * @return Task|void
  */
-function fail(string $task, string $do)
+function fail(string $task, $do)
```

Comment:
in this line a bug was fixed. `fail` also accepts a callable, and therefore the native type was wrong.

Reply:
should I add a changelog entry for this fix?